spirekey/src/utils/getNetworkDisplayName.ts

```diff
@@ -4,3 +4,8 @@ export const getNetworkDisplayName = (network: string) => {
   if (network === 'development' || 'fast-development') return 'Devnet';
   return network;
 };
+
+export const getRootkeyPasskeyName = (network: string) => {
+  const rootkeyPasskeyName = `SpireKey Wallet Manager (DO NOT USE FOR SIGNING)`;
```

Comment:
nit: I have a preference for `SpireKey Wallet Manager` over `SpireKey Wallet Manager (DO NOT USE FOR SIGNING)` because I feel like the name already indicates that it is not a normal key, it won't work if the user selects it, and most of all it doesn't look as pretty haha. But don't mind keeping it if you think it adds value

Reply:
Kadena Spirekey Wallet Manager